src/runtime/browser/base64url.ts

```diff
@@ -0,0 +1,41 @@
+const encoder = new TextEncoder();
+const decoder = new TextDecoder();
+
+export const encodeBase64 = (input: Uint8Array | string) => {
```

Comment:
@ragnika I see this implementation is borrowed from `jose` library. Does it handle utf-8?

Reply:
@linasi yes. TextEncoder emits a stream of utf-8 bytes

https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder